Assets/Scripts/FlightScripts/Spaceship.cs

```diff
@@ -17,6 +17,7 @@ public class Spaceship : MonoBehaviour
         private float _horizontalOffset;
 
         public float Speed { get; private set; }
+        public float HorizontalOffset { get { return _horizontalOffset; } set { _horizontalOffset = value; } }
```

Comment:
Kannst des noch direkt assignen? Also:
```suggestion
        public float HorizontalOffset { get; set; }
```